### ISSUE TITLE
AttributeError: 'NoneType' object has no attribute 'Response'

### ISSUE THREAD

**Issue by yingmu52** [jmagnusson/netsuite#18]
```
Traceback (most recent call last):
  File "/usr/local/bin/netsuite", line 5, in <module>
    from netsuite.__main__ import main
  File "/usr/local/lib/python3.7/site-packages/netsuite/__init__.py", line 4, in <module>
    from .client import NetSuite  # noqa
  File "/usr/local/lib/python3.7/site-packages/netsuite/client.py", line 19, in <module>
    from .rest_api import NetSuiteRestApi
  File "/usr/local/lib/python3.7/site-packages/netsuite/rest_api.py", line 38, in <module>
    class NetSuiteRestApi:
  File "/usr/local/lib/python3.7/site-packages/netsuite/rest_api.py", line 139, in NetSuiteRestApi
    ) -> httpx.Response:
AttributeError: 'NoneType' object has no attribute 'Response'
```

I am getting this error, and I am not sure why.

**Comment by wwatkinson** [comment]
@yingmu52 this is due to an issue with a new file in the latest version of the package. I recommend moving back to version 0.4.1 of the package.

@jmagnusson FYI the latest version of the package is unusable due to the try/except around the import httpx statement meaning that sometimes httpx = None, which breaks the httpx.Response type hint on line 140.

**Comment by jmagnusson** [comment]
@yingmu52 @wwatkinson This should be fixed in 0.5.3. Available on PyPI now.
